src/warnet/k8s.py

```diff
@@ -245,3 +245,27 @@ def wait_for_caddy_ready(name, namespace, timeout=300):
                 return True
     print(f"Timeout waiting for Caddy pod {name} to be ready.")
     return False
+
+
+def get_kubeconfig_value(jsonpath):
+    command = f"kubectl config view --minify -o jsonpath={jsonpath}"
```

Comment:
seems a little silly to me to import the kubernetes python client library into this module and then just use shell commands anyway, but the whole file is like that so it should just be a cleanup PR later

Reply:
Heh, I had the same thought. I am somewhat in favour of using the Kubernetes python client for everything, but noticed everyone was defaulting to string-ifying `kubectl` commands. So long as we keep all the k8s logic in this file, should be super easy to refactor this at some point in the future.